src/generated/resources/data/tconstruct/tinkering/modifiers/melee_protection.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "tconstruct:composable",
+  "level_display": "tconstruct:default",
+  "modules": [
+    {
+      "type": "tconstruct:max_armor_stat",
+      "allow_broken": false,
+      "each_level": 0.005,
+      "key": "tconstruct:use_item_speed",
+      "key2": "tconstruct:melee_protection"
```

Comment:
These are not good names for the fields. Its very unclear in the JSON what either does